GettingStarted/GettingStarted.md

```diff
@@ -20,14 +20,18 @@ Getting in to NCSA takes involves getting an NCSA account, and then figuring out
 Contact Phil (DM @drphilmarshall on LSSTC Slack) or Simon (@ksk) to get an NCSA Stack Club account. You'll need to provide your full name (first and last) and your email address. You'll (eventually) get an email invitation to [create an account at NCSA](https://identity.ncsa.illinois.edu/) (including choosing a username of 8 characters or fewer). After you have submitted your form, it typically takes 24 hours for your account to be set up: set an alarm to come back the next day!
 
 ### Accessing NCSA via its VPN
-As explained on the [PDAC wiki](https://confluence.lsstcorp.org/display/DM/PDAC+networking+and+user+accounts+for+developers), the only way to get onto the LSST science platform at NCSA is via its virtual private network (VPN). The easiest way is by using Cisco's AnyConnect, which if you don't have you can get by pointing your browser at **https://vpn.ncsa.illinois.edu/** and selecting the `ncsa-vpn-default` option (this will only work if you have a java-compatible browser, like firefox esr version<=52). If you already have the AnyConnect client installed, open it up and enter `vpn.ncsa.illinois.edu` in its connection window. 
+As explained on the [PDAC wiki](https://confluence.lsstcorp.org/display/DM/PDAC+networking+and+user+accounts+for+developers), the only way to (currently) get onto the LSST science platform at NCSA is via the [NCSA virtual private network (VPN)](https://wiki.ncsa.illinois.edu/display/cybersec/Virtual+Private+Network+%28VPN%29+Service). The easiest way is by using Cisco's AnyConnect with DUO two-factor authentication. 
```

Comment:
The PDAC as described in that reference is a separate system from the lsst-lspdev deployment, and there's a lot of detail on that page that doesn't apply to the notebook deployment we are using. I recommend instead:

At present, unless you are on an approved network, you must use the [NCSA virtual private network (VPN)]((https://wiki.ncsa.illinois.edu/display/cybersec/Virtual+Private+Network+%28VPN%29+Service).
The recommended method is to use Cisco's AnyConnect with DUO two-factor authentication.

Reply:
(when https://nb.lsst.io/getting-started/logging-in.html is updated I feel we should link to it)